core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/textannotation/View.java

```diff
@@ -559,4 +558,9 @@ public int count() {
         return this.constituents.size();
     }
 
+    @Override
+    public Object clone() throws CloneNotSupportedException {
+        View view = (View) super.clone();
```

Comment:
Question: Shouldn't cloning a view also clone its constituents and relations respectively? Also is the cloned View attached to the same TextAnnotation object? 

I don't understand how cloning Views and Constituents independent of their TextAnnotation would work?